core/templates/dev/head/player/ConversationSkinDirective.js

```diff
@@ -247,11 +247,16 @@ oppia.animation('.conversation-skin-animate-card-contents', function() {
   };
 });
 
-oppia.directive('conversationSkin', [function() {
+oppia.directive('conversationSkin', ['urlService', function(urlService) {
   return {
     restrict: 'E',
     scope: {},
-    templateUrl: 'skins/Conversation',
+    link: function(scope) {
+      var isIframed = urlService.isIframed();
+      scope.myTemplate = isIframed ?
+        'skins/ConversationEmbed' : 'skins/Conversation';
+    },
+    template: "<div ng-include='myTemplate'></div>",
```

Comment:
Single quotes on outside, double quotes in HTML tags. Viz.:

'<div ng-include="..."></div>'